### PR TITLE
Create event styling

### PR DESCRIPTION
**Pull request by Boschio**
Worked on styling for create event form. Added logic for added/removing searched games.

### CODE REVIEW

File: gameon-api/models/posts 2.js [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+class Posts {
+    static async createNewPost({post, user, eventId}) {
+        const requiredFields = ["postTitle", "postContent"]
+        requiredFields.forEach((field) => {
+            if (!post.hasOwnProperty(field) || !post[field]) {
+                throw new BadRequestError(`Required field - ${field} - missing from request body.`)
+            }
+        })
+        console.log("user: ", user)
```
Comment (Rebecca-CYLi) [inline gameon-api/models/posts 2.js:13]: console logs?

File: gameon-api/models/posts 3.js [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+class Posts {
+    static async createNewPost({post, user, eventId}) {
+        const requiredFields = ["postTitle", "postContent"]
+        requiredFields.forEach((field) => {
+            if (!post.hasOwnProperty(field) || !post[field]) {
+                throw new BadRequestError(`Required field - ${field} - missing from request body.`)
+            }
+        })
+        console.log("user: ", user)
```
Comment (Rebecca-CYLi) [inline gameon-api/models/posts 3.js:13]: console logs?

File: gameon-api/routes/games 2.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
+const express = require("express")
+const router = express.Router()
+const axios = require("axios")
+const { TWITCH_CLIENT_ID, TWITCH_APP_ACCESS_TOKEN } = require("../config");
+const Games = require("../models/games");
+
+router.post("/", async (req, res, next) => {
+    const searched = req.body
+    console.log("searched: ", searched.searched)
```
Comment (Rebecca-CYLi) [inline gameon-api/routes/games 2.js:9]: console logs?

File: gameon-api/routes/games 2.js [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+        })
+        .catch((err) => {
+            console.error(err);
+        });
+});
+
+router.post("/id", async (req, res, next) => {
+    const gameId = req.body
+    let game
+    console.log("gameId: ", gameId)
```
Comment (Rebecca-CYLi) [inline gameon-api/routes/games 2.js:32]: console logs

File: gameon-api/routes/posts 2.js [hunk 5]
```diff
@@ -0,0 +1,29 @@
+const express = require("express")
+const Posts = require("../models/posts")
+const security = require("../middleware/security")
+const router = express.Router()
+
+router.post("/:eventId", async (req, res, next) => {
+    try {
+        // create a new event
+        const { user } = res.locals
+        console.log("user (routes): ", user)
```
Comment (Rebecca-CYLi) [inline gameon-api/routes/posts 2.js:10]: console logs?

File: gameon-api/routes/posts 2.js [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+        const { eventId } = req.params
+        console.log("req.body: ", req.body)
```
Comment (Rebecca-CYLi) [inline gameon-api/routes/posts 2.js:12]: console logs?

File: gameon-api/routes/posts 3.js [hunk 7]
```diff
@@ -0,0 +1,29 @@
+const express = require("express")
+const Posts = require("../models/posts")
+const security = require("../middleware/security")
+const router = express.Router()
+
+router.post("/:eventId", async (req, res, next) => {
+    try {
+        // create a new event
+        const { user } = res.locals
+        console.log("user (routes): ", user)
```
Comment (Rebecca-CYLi) [inline gameon-api/routes/posts 3.js:10]: console logs?

File: gameon-api/routes/posts 3.js [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+        const { eventId } = req.params
+        console.log("req.body: ", req.body)
```
Comment (Rebecca-CYLi) [inline gameon-api/routes/posts 3.js:12]: console logs?

**Review by Rebecca-CYLi — CHANGES_REQUESTED** [review]
remove all console logs

**Comment by LifeBringer** [comment]
(just noting this PR, please ignore this comment)

File: gameon-api/models/posts 2.js (hunk 1 above)
Comment (Boschio) [reply gameon-api/models/posts 2.js]: Removed as requested

**Review by Rebecca-CYLi — APPROVED** [review]
removed unwanted files & console logs

LGTM

**Comment by Boschio** [comment]
This needs to be look at once more after further testing. Functionality is not working as intended. Will have Charidi dive back in and take a look.

**Pull request closed without merging**